brianlong/www.validators.app
# 179040897_move_stats_from_homepage_to_cluster_stats_view

#### What's this PR do?
- Move stats to cluster stats view

#### How should this be manually tested?
- Go to the footer, click 'Cluster Stats', scroll to the bottom of the page, check the 'General stats'

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/179040897

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
